AzureReverseProxy/AzureReverseProxy/AzureReverseProxyCMD/Program.cs

```diff
@@ -0,0 +1,39 @@
+using System;
```

Comment:
Add Copyright header

Reply:
Disregard code-related as those changes are implemented in the branch yet to be merged with this one.